packages/raknet/src/packets/unconnected-pong.ts

```diff
@@ -0,0 +1,30 @@
+import { Packet } from './packet'
+import { BinaryStream } from '@jukebox/binarystream'
+
+export class UnconnectedPong extends Packet {
```

Comment:
*Ping (idk)

Seguendo il commento sopra devi anche implementare `IPacket`
```suggestion
export class UnconnectedPong extends Packet implements IPacket {
```